src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -478,11 +510,41 @@ export default Vue.extend({
           this.player.volume(0)
         }
 
-        if (!this.player.muted()) {
```

Comment:
Just in case anyone is wondering about this, I had to modify the mouseSrollVolume method in order to make sure that the ctrl/command ket was NOT pressed so that a simple scroll will not change the volume AND the playback rate